src/redux/reducers/messages.js

```diff
@@ -119,6 +119,11 @@ export default handleActions(
         messagesByID,
       };
     },
+    [actionTypes.APP_LOGOUT](state, { payload }) {
```

Comment:
@akshatnitd Not sure if this is what you meant by cleaning app the message store. To be honest this is redundant here but I think for the sake of code consistency, this should work well.